popupWindow.js

```diff
@@ -0,0 +1,157 @@
+const modalContainer = document.querySelector('.modal-container');
+const modal = document.querySelector('.modal');
+// var to match if the screen is for mobile or for desktop
+const x = window.matchMedia('(min-width: 768px)'); // desktop
+// array to store element
+const projectList = {
+  project1: {
+    title: 'Keeping track of hundreds of components',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featured_image: new Image(),
+    technologies: ['Ruby on rails', 'css', 'Javascript'],
+    technologiesDesktop: [
+      'Codekit',
+      'GitHub',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    liveBtnImge: new Image(),
+    SourceBtn: new Image(),
+  },
+};
```

Comment:
- [ ] You should have objects in the array. Each object represents a single project. Please add six objects at least :+1:
- [ ] Your Popup is not working as mentioned by the previous reviewer. Please fix it. :+1:

Hint: You can see this [example](https://github.com/HAMMAS-SALEEM/Portfolio/blob/master/script/app.js) to get an idea of how you are gonna do this :blush:

Reply:
fix the popup

Reply:
> * [ ]  You should have objects in the array. Each object represents a single project. Please add six objects at least +1
> * [x]  Your Popup is not working as mentioned by the previous reviewer. Please fix it. +1
> 
> Hint: You can see this [example](https://github.com/HAMMAS-SALEEM/Portfolio/blob/master/script/app.js) to get an idea of how you are gonna do this blush

- [ ] As the previous reviewers suggested, Kindly work on the Popup design.

**Mobile view:**

Your design
:-------------------------:
![designmobile](https://user-images.githubusercontent.com/70483835/189347244-0818ce17-b850-4fab-9e31-6db1b1aa2310.png)

Figma Design
:-------------------------:
![figmamobileview](https://user-images.githubusercontent.com/70483835/189347216-600b7f86-435a-4655-b42a-aaf4ca032de0.png)

**Desktop view:**

Your design
:-------------------------:
![designdesktopview](https://user-images.githubusercontent.com/70483835/189347496-90837799-42ff-43a7-a95e-06f31334694e.png)

Figma Design
:-------------------------:
![figma desktop view](https://user-images.githubusercontent.com/70483835/189347681-9515b17d-0937-45f1-9e92-8b00b6970afc.png)

Reply:
update the padding and margin